packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -752,6 +752,7 @@ function performConcurrentWorkOnRoot(root, didTimeout) {
       return null;
     } else {
       // Current task was not canceled. Continue.
+      originalCallbackNode = root.callbackNode;
```

Comment:
What am I missing? 

This seems semantically the same since we only set it if `root.callbackNode === originalCallbackNode`

Reply:
Sigh you're not missing anything, I pushed the wrong commit 🤦‍♂️